google-cloud-storage/src/main/java/com/google/cloud/storage/StorageOptions.java

```diff
@@ -38,8 +38,8 @@
   private static final String GCS_SCOPE = "https://www.googleapis.com/auth/devstorage.full_control";
   private static final Set<String> SCOPES = ImmutableSet.of(GCS_SCOPE);
   private static final String DEFAULT_HOST = "https://storage.googleapis.com";
-
   private final RetryAlgorithmManager retryAlgorithmManager;
+  private final Boolean invocationIDEnabled;
```

Comment:
Since we're always going to have a value, we can use `boolean` throughout for this flag. `Boolean` is really only useful if you need to deal with `true|false|null`.